pkg/sbproxy/notifications/handlers/broker_handler.go

```diff
@@ -90,6 +94,11 @@ func (bnh *BrokerResourceNotificationsHandler) OnCreate(ctx context.Context, pay
 	brokerProxyPath := bnh.brokerProxyPath(brokerToCreate.Resource)
 	brokerProxyName := bnh.brokerProxyName(brokerToCreate.Resource)
 
+	if slice.StringsAnyEquals(bnh.BrokerBlacklist, brokerToCreate.Resource.Name) {
+		log.C(ctx).Infof("Broker with name %s is part of broker blacklist. Skipping notification...", brokerToCreate.Resource.Name)
```

Comment:
what about notifications for visibilities for plans of brokers that are blacklisted ? They will come too.

Reply:
Enable and disable access will fail as the broker for which access will try to be enabled/disabled won't exist (for both Create/Update vis notifications):

https://github.com/Peripli/service-broker-proxy/blob/aac6c78095f2d61a4936cfbebf11964668b1b732/pkg/sbproxy/notifications/handlers/visibilities_handler.go#L98

https://github.com/Peripli/service-broker-proxy/blob/aac6c78095f2d61a4936cfbebf11964668b1b732/pkg/sbproxy/notifications/handlers/visibilities_handler.go#L136

Reply:
Essentially, nothing should mess up, but we'll get some failure logs. I can check whether the broker for which access will be created/changed is in the blacklist or not?

Reply:
Yes i know that we will get failure logs. The comment implied that in my opinion we should also handle /skip these notifications as we don;'t want false positive errors - maybe i did not form my comment well enough :)

Reply:
Alright, I'll handle it for both the broker and visibilities then

Reply:
Addressed.